content/sensu-go/5.14/getting-started/learn-sensu.md

```diff
@@ -36,38 +36,37 @@ ENABLE_SENSU_SANDBOX_PORT_FORWARDING=1 vagrant up
 {{< /highlight >}}
 
 The Learn Sensu sandbox is a CentOS 7 virtual machine pre-installed with Sensu, InfluxDB, and Grafana.
-It is intended for use as a learning tool; we do not recommend this tool as part of a production installation.
-To install Sensu in production, please see the [installation guide](../../installation/install-sensu).
-The sandbox startup process takes about five minutes.
+It's intended for you to use as a learning tool--we do not recommend using it in a production installation. To install Sensu in production, use the [installation guide](../../installation/install-sensu) instead.
 
-_NOTE: The sandbox configures VirtualBox to forward TCP ports 3002 and 4002 from the sandbox virtual machine to the localhost to make it easier for you to interact with the sandbox dashboards. Dashboard links provided in this tutorial assume port forwarding from the VM to the host is active._
+The sandbox startup process takes about 5 minutes.
+
+_NOTE: The sandbox configures VirtualBox to forward TCP ports 3002 and 4002 from the sandbox virtual machine (VM) to the localhost to make it easier for you to interact with the sandbox dashboards. Dashboard links provided in this tutorial assume port forwarding from the VM to the host is active._
 
 **4. SSH into the sandbox**
 
-Thanks for waiting! To start using the sandbox:
+Thanks for waiting! To start shell access to the sandbox:
 
 {{< highlight shell >}}
 vagrant ssh
 {{< /highlight >}}
 
-You should now have shell access to the sandbox and should be greeted with this prompt:  
+You should be greeted with this prompt:  
 
 {{< highlight shell >}}
 [sensu_go_sandbox]$
 {{< /highlight >}}
 
-To exit out of the sandbox, use `CTRL`+`D`.
-To erase and restart the sandbox, use `vagrant destroy` then `vagrant up`.
-To reset the sandbox's Sensu configuration to the beginning of this tutorial, use `vagrant provision`.
+- To exit out of the sandbox, press `CTRL`+`D`.
+- To erase and restart the sandbox, run `vagrant destroy` and then `vagrant up`.
+- To reset the sandbox's Sensu configuration to the beginning of this tutorial, run `vagrant provision`.
 
-_NOTE: The sandbox pre-configures sensuctl with the Sensu Go admin user, so you won't have to configure sensuctl each time you spin up the sandbox to try out a new feature. Before installing sensuctl outside of the sandbox, read the [first time setup reference](https://docs.sensu.io/sensu-go/5.0/sensuctl/reference/#first-time-setup) to learn how to configure sensuctl._  
+_NOTE: The sandbox pre-configures sensuctl with the Sensu Go admin user, so you won't have to configure sensuctl each time you spin up the sandbox to try out a new feature. Before installing sensuctl outside of the sandbox, read the [first time setup reference][1] to learn how to configure sensuctl._  
```

Comment:
For some reason the link to `[1]` still throws me to the 5.0 release (the original link). I do see that isn't the case down below, so might be a cache thing for me. If no one else has that issue, let's not worry about it.

Reply:
🤦‍♂ . I had the wrong branch checked out locally. Disregard.